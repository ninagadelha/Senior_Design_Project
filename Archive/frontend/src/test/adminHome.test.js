import renderer from "react-test-renderer";
import AdminHome from "../routes/adminHome";

jest.mock("../components/AdminNavBar");
jest.mock("../components/footer");
jest.mock("../styles/adminHome.css");
jest.mock("react-router-dom");

const renderTree = (tree) => renderer.create(tree);
describe("<AdminHome>", () => {
  it("should render component", () => {
    expect(renderTree(<AdminHome />).toJSON()).toMatchSnapshot();
  });
});
