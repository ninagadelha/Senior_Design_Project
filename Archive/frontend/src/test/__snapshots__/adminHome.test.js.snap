// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminHome> should render component 1`] = `
<div>
  <main>
    <section
      className="intro"
    >
      <h2>
        Welcome to MySTEMGrowth Admin Profile!
      </h2>
    </section>
    <section
      className="tool-description"
    >
      <p>
        This tool will help students to understand their beliefs about their STEM abilities. This inventory will capture a snapshot that will generate a profile that students can use to recognize their STEM growth potential.
      </p>
    </section>
    <section
      className="adminInfo"
    >
      <h4>
        Access the following pages to view student's results, create new programs, or make changes to the survey questions!
      </h4>
    </section>
  </main>
  <div
    className="container"
  >
    <div>
      <div>
        <div
          className="content-placeholder"
        />
      </div>
    </div>
  </div>
</div>
`;
